Tools/CISupport/ews-build/steps.py

```diff
@@ -6569,8 +6572,12 @@ def _files(self):
     def run(self, BufferLogObserverClass=logobserver.BufferLogObserver):
         base_ref = self.getProperty('github.base.ref', f'{DEFAULT_REMOTE}/{DEFAULT_BRANCH}')
         head_ref = self.getProperty('github.head.ref', 'HEAD')
-        reviewers = self.getProperty('reviewers_full_names', None)
```

Comment:
`reviewers_full_names` property is renamed in this PR, but all the places which use this property doesn't seem to be updated. This would break Merge-Queue (e.g.: AddReviewerToCommitMessage would stop adding reviewer name to commit message and then ValidateCommitMessage will fail).

Reply:
Great catch! I meant to do that but forgot.. I'm also going to be testing the changes on DEV today.

Reply:
Here's an example of AddReviewerToCommitMessage working as intended: https://ews-build.webkit-dev.org/#/builders/38/builds/35825

Reply:
Example of no valid reviewer: https://ews-build.webkit-dev.org/#/builders/38/builds/35826